src/components/ActionButton.tsx

```diff
@@ -0,0 +1,18 @@
+type ActionButtonProps = {
+    label: string,
+    onClick?: any,
```

Comment:
nit: ideally, we should avoid using "any" types. You should be able to use the following:

```
onClick?: () => void;
```